patches/minecraft/net/minecraft/item/ItemBow.java.patch

```diff
@@ -21,11 +21,47 @@
                  if ((double)f >= 0.1D)
                  {
 -                    boolean flag1 = flag && itemstack.func_77973_b() == Items.field_151032_g;
-+                    boolean flag1 = flag && itemstack.func_77973_b() instanceof ItemArrow; //Forge: Fix consuming custom arrows.
++                	boolean isArrowInfinite = determineIsArrowInfinite(flag, itemstack);
```

Comment:
Excellent, that's an extremely helpful resource.  Was looking for something like that and so many things are outdated.  To be clear, I have no problem changing to spaces, was just curious if there was a technical reason for it.  Thanks for the input, much appreciated.